examples/MQ/pixelSimSplit/src/devices/FairMQChunkMerger.cxx

```diff
@@ -144,6 +146,7 @@ bool FairMQChunkMerger::MergeData(fair::mq::Parts& parts, int /*index*/)
                 continue;   // MCTrack already done, renumber all _other_ arrays...
             //            LOG(info) << "BEFORE ADDING, TCA \"" << tca_elem->GetName() << "\" has " <<
             //            tca_elem->GetEntries() << " entries.";
+            fRet = fObjectMap.equal_range(fEvRIPair);
             int addedArray = 0;
             TClonesArray* arrayToAdd;
```

Comment:
The analyzer is still not happy because this variable may stay uninitialized. Can you remove it from this scope entirely and declare-and-initialize it locally inside the if block (and also move the `tca_elem->AbsorbObjects(arrayToAdd)` inside the if block)?

Reply:
sure, if your goal is to just silence the compiler warnings, but this is supposed to be example code and imho it deserves more care.